orchagent/routeorch.cpp

```diff
@@ -371,6 +374,7 @@ bool RouteOrch::invalidnexthopinNextHopGroup(const NextHopKey &nexthop)
             return false;
         }
 
+        ++ count;
```

Comment:
what is the purpose of this "count" variable? Does not look like it is being used?

Reply:
This is the return value as part of function param.